libafl/src/corpus/ondisk.rs

```diff
@@ -213,14 +210,8 @@ where
             let mut ctr = 2;
             let filename = loop {
                 let lockfile = format!(".{file}.lafl_lock");
-                // try to create lockfile.
-
-                if OpenOptions::new()
-                    .write(true)
-                    .create_new(true)
-                    .open(self.dir_path.join(lockfile))
-                    .is_ok()
-                {
+
+                if !Path::new(&lockfile).exists() {
```

Comment:
This won't create the lockfile anymore, so it'll be racy again, if some other process accesses the lockfile at the same time, right?

Reply:
Hi domenukk I've fixed this.
That is why  the lock file is create lock  testcase.rs on the first place instead of inside ondisk.rs.
I've reverted  the logic related old testcase lock file back to testcase.rs